src/WidgetBase.ts

```diff
@@ -209,6 +211,10 @@ export class WidgetBase<P extends WidgetProperties = WidgetProperties, C extends
 
 	private _renderState: WidgetRenderState = WidgetRenderState.IDLE;
 
+	private _metaMap = new WeakMap<WidgetMetaConstructor<any>, MetaBase>();
```

Comment:
Do you think we should add a line break between these like all the other class properties?